query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -33,6 +33,7 @@ impl ToSqlRow for ResultRow {
     fn to_sql_row<'b>(self, idents: &[(TypeIdentifier, FieldArity)]) -> crate::Result<SqlRow> {
         let mut row = SqlRow::default();
         let row_width = idents.len();
+        row.values.reserve(row_width);
```

Comment:
Hmm. What is the difference between this and then implementing `with_capacity` to `SqlRow` and using it?

Reply:
I think it's the same.